pkg/imgpkg/bundle/bundle.go

```diff
@@ -86,3 +151,9 @@ func (o *Bundle) checkedImage() (regv1.Image, error) {
 	}
 	return img, err
 }
+
+type noopWriter struct{}
+
+func (noopWriter) Write(_ []byte) (n int, err error) {
+	return 0, nil
```

Comment:
len(bs) should be returned instead of 0 to follow io.Writer interface.

Reply:
This struct was deleted in favor of using the `goui.NewNoopUI`